src/Seld/JsonLint/JsonParser.php

```diff
@@ -326,6 +332,7 @@ public function parse($input, $flags = 0)
             }
         }
 
+        // @phpstan-ignore-next-line
```

Comment:
Why is this needed? Is true not part of mixed?!

Reply:
Phpstan thinks this is unreachable code

Reply:
I guess technically it is, there doesn't seem to be a way out of the while (true) above, except for the returns, so I don't think this is needed.

Reply:
so would you agree we throw a exception in case this path is taken?

Reply:
Yeah sounds good, if the tests pass :D

Reply:
or just remove the statement entirely, if this path cannot be taken anyway.

Reply:
phpstan also complained about the `throw Exception` line to be never reachable.. just deleted it, as suggested by stof.